TheWidlarzGroup/react-native-video
# docs: update DRM token generator links

## Summary
This PR updates the example app in the react-native-video library with functioning link to generate DRM token. The placeholders for these links have been replaced with working URLs, allowing users to generate and use DRM tokens effectively.

### Motivation

### Changes
Replaced all placeholder links with actual URLs pointing to the DRM token generator.
## Test plan
- Open the example app and follow the updated instructions to generate a DRM token using the provided link.
- Replace <USER_TOKEN> in the code with the generated token.
- Confirm that the video plays correctly with the new token in place.
